src/js/firstparties/twitter.js

```diff
@@ -80,6 +122,33 @@ function unwrapSpecialTwitterURLs() {
   });
 }
 
-unwrapSpecialTwitterURLs();
-unwrapTwitterURLsInTimeline();
-setInterval(unwrapTwitterURLsInTimeline, 2000);
+function unwrapSpecialTwitterPwaURLs() {
+  // profile URL can be found in JSON of page
+
+  try {
+    // iterate users object (usually should only be one, with some random ID?)
+    for (const entityKey of Object.keys(window.wrappedJSObject.__INITIAL_STATE__.entities.users.entities)) {
```

Comment:
`wrappedJSObject` is Firefox-only. Getting errors in Chrome:
```
TypeError: Cannot read property '__INITIAL_STATE__' of undefined
    at unwrapSpecialTwitterPwaURLs (twitter.js:130)
    at twitter.js:145
```

Reply:
If there isn't a straightforward cross-browser way to grab the profile link, we might want to just skip profile link replacement on the Twitter PWA.

Reply:
Okay, so what is the chrome equivalent?

Reply:
I don't think there is a direct replacement, and I'm not sure it's worth complicating this script just for the profile link.

Can we get the full URL from the title attribute, same as on regular Twitter? Is the problem that we don't have a stable-seeming selector to target the profile area? But querying for all http/https t.co href attributes with http/https title attributes should pick that link up too, no?

Reply:
now just check for `wrappedJSObject` (feature detection), and ignore it if it is not needed